app/controllers/admin/transactions_controller.rb

```diff
@@ -0,0 +1,22 @@
+class Admin::TransactionsController < Admin::AdminController
+
+  def index
+    authorize :transaction, :index?
+
+    auth_details = {access_token: Division.connector.access_token, company_id: Division.connector.realm_id}
```

Comment:
I suggest adding a private helper method `qb_connection` that memoizes `Division.root.qb_connection`. Then you can use that throughout.